source/connection-monitoring-and-pooling/tests/README.rst

```diff
@@ -26,11 +26,4 @@ The following tests have not yet been automated, but MUST still be tested:
 #. All ConnectionPoolOptions MUST be the same for all pools created by a MongoClient
 #. A user MUST be able to specify all ConnectionPoolOptions via a URI string
 #. A user MUST be able to subscribe to Connection Monitoring Events in a manner idiomatic to their language and driver
-#. When a check out attempt fails because connection set up throws an error,
-   assert that a ConnectionCheckOutFailedEvent with reason="connectionError" is emitted.
 
-Logging Tests
-=============
-
-Tests for connection pool logging can be found in the `/logging <./logging>`__ subdirectory and are written in the 
-`Unified Test Format <../../unified-test-format/unified-test-format.rst>`__. 
```

Comment:
Was this intentionally removed?

Reply:
Yep, the logging and new unified tests were combined into a single `unified` directory, rather than having a separate one for logging.